### ISSUE TITLE
Grouped Window List applet: hotkey produces array of tooltips

### ISSUE THREAD

**Issue by LinuxOnTheDesktop** [linuxmint/cinnamon#8633]
```
 * Cinnamon version (cinnamon --version) 4.0.10
   - Please specify if you are using the daily builds PPA (https://launchpad.net/~linuxmint-daily-build-team/+archive/ubuntu/daily-builds). yes
 * Distribution - (Mint 17.2, Arch, Fedora 25, etc...) Mint 19.1
 * Graphics hardware *and* driver used: Intel UHD 620 Mesa 3
 * 32 or 64 bit: 64
 * Attach /home/<username>/.xsession-errors, or /var/log/syslog
 ```

**Issue**

Using a particular keyboard shortcut (see below for the combination) to cycle through the previews produces this:

![image](https://user-images.githubusercontent.com/47056543/59321230-2e067100-8cc9-11e9-8dea-2311f4a6e30c.png)

Note: I have the panel set to autohide.

**Steps to reproduce**

Assign this key combination to cycling through thumbnail windows: ctrl-alt-right arrow. (Other key combinations do not have the effect at issue.)

**Expected behaviour**

Cycling through window list - as does happen with other key combinations.

**Other information**

**Comment by clefebvre** [comment]
Does it happen with 4.0.10 (i.e. non-ppa?).

**Comment by LinuxOnTheDesktop** [comment]
_No_ (I tested on virtual box on another computer).

_Indeed_: I have been unable, even on the alpha, to recreate the problem. 

_Yet_, on the alpha, I did find the the following. Across a range of Cinnamon themes, the ctrl-alt-right key combination generated a range of strange behaviours, including a simple failure to switch from one panel preview to another.

Also, I discovered another minor problem, for which I have created [another bug report](https://github.com/linuxmint/cinnamon/issues/8634).
